app/auth/middleware.js

```diff
@@ -1,5 +1,5 @@
 const User = require('../../common/lib/user')
-const userLocationService = require('../../common/services/user-locations')
+const { getUserLocations } = require('../../common/services/user-locations')
```

Comment:
Might want to make sure this isn't affecting the test stubs. 

I came across problems when stubbing methods that were destructured in the initial module.